MergerService/Runners/TaskExecutor.cs

```diff
@@ -162,6 +162,7 @@ public void ExecuteTask(MergeTask task, ITaskUtils taskUtils, string? managerCal
                                     List<CorrespondingTileBuilder> correspondingTileBuilders = new List<CorrespondingTileBuilder>();
                                     // Add target tile
                                     correspondingTileBuilders.Add(() => getTileByCoord(sources[0], coord));
```

Comment:
remove irrelevent definition of (_) => null

Reply:
Removed.